src/helm/benchmark/scenarios/clear_scenario.py

```diff
@@ -42,28 +42,71 @@ class CLEARScenario(Scenario):
         Answer:
     """
 
-    name = "clear_alcohol_dependence"
-    description = (
-        "A dataset for evaluating alcohol dependence history "
-        "detection from patient notes with yes/no/maybe classifications."
-    )
-    tags = ["classification", "biomedical"]
-
-    POSSIBLE_ANSWER_CHOICES: List[str] = [
-        "Has a history of alcohol dependence",
-        "Does not have a history of alcohol dependence",
-        "Uncertain",
+    # List of all available conditions
+    CONDITIONS = [
+        "alcohol_dependence",
+        "attention_deficit_hyperactivity_disorder",
+        "bipolar_disorder",
+        "chronic_pain",
+        "homelessness",
+        "liver_disease",
+        "major_depression",
+        "personality_disorder",
+        "post_traumatic_stress_disorder",
+        "substance_use_disorder",
+        "suicidal_behavior",
+        "tobacco_dependence",
+        "unemployment",
     ]
 
+    # Map condition names to human-readable descriptions for prompts
+    CONDITION_PROMPTS = {
+        "alcohol_dependence": "alcohol dependence",
+        "attention_deficit_hyperactivity_disorder": "attention deficit hyperactivity disorder (ADHD)",
+        "bipolar_disorder": "bipolar disorder",
+        "chronic_pain": "chronic pain",
+        "homelessness": "homelessness",
+        "liver_disease": "liver disease",
+        "major_depression": "major depression",
+        "personality_disorder": "personality disorder",
+        "post_traumatic_stress_disorder": "post-traumatic stress disorder (PTSD)",
+        "substance_use_disorder": "substance use disorder",
+        "suicidal_behavior": "suicidal behavior",
+        "tobacco_dependence": "tobacco dependence",
+        "unemployment": "unemployment",
+    }
+
+    def __init__(self, condition: str):
+        """Initialize the scenario with a specific medical condition"""
+        super().__init__()
+
+        if condition not in self.CONDITIONS:
+            raise ValueError(f"Condition '{condition}' not supported. Available conditions: {self.CONDITIONS}")
+
+        self.condition = condition
+        self.name = f"clear_{condition}"
+        self.description = f"A dataset for evaluating {self.CONDITION_PROMPTS[condition]} detection from patient notes with yes/no/maybe classifications."
```

Comment:
Add `  # noqa: E501` at the end of this line to make the linter happy.

Reply:
Thank you! Fixed!